ha/setup/ha_setup.py

```diff
@@ -848,16 +848,16 @@ def process(self):
                 # TODO: Update cluster kill for --local option also
                 # Remove SSH
                 self._remove_node(node_name)
+                if self._confstore.key_exists(f"{const.CLUSTER_CONFSTORE_NODES_KEY}/{node_name}"):
+                    self._confstore.delete(f"{const.CLUSTER_CONFSTORE_NODES_KEY}/{node_name}")
+                # Delete node name mapping from store
+                if self._confstore.key_exists(f"{const.PVTFQDN_TO_NODEID_KEY}/{node_name}"):
+                    self._confstore.delete(f"{const.PVTFQDN_TO_NODEID_KEY}/{node_name}")
```

Comment:
Can we delete node map stored in consul as well?

Reply:
Updated